public/assets/main.css

```
@@ -7800,9 +7717,9 @@ table.dataTable thead td {
 .disabled\:opacity-60:disabled {
   opacity: 0.6;
 }
-.disabled\:hover\:bg-green-600:hover:disabled {
+.disabled\:hover\:bg-gray-800:hover:disabled {
   --tw-bg-opacity: 1;
-  background-color: rgb(22 163 74 / var(--tw-bg-opacity, 1));
+  background-color: rgb(31 41 55 / var(--tw-bg-opacity, 1));
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)